networkit/cpp/graph/test/GraphGTest.cpp

```diff
@@ -2381,5 +2381,21 @@ TEST_P(GraphGTest, testSubgraphFromNodesDirected) {
 
 }
 
+TEST_P(GraphGTest, testRemoveMultiEdges) {
+	Aux::Random::setSeed(42, false);
+	Graph G(this->Ghouse);
+	const count nMultiEdges = 10;
+	const count m = G.numberOfEdges();
+
+	// Adding multiedges at random
+	for (count i = 0; i < nMultiEdges; ++i) {
+		auto e = G.randomEdge();
+		G.addEdge(e.first, e.second);
+	}
+
+	EXPECT_EQ(G.numberOfEdges(), m + nMultiEdges);
+	G.removeMultiEdges();
+	EXPECT_EQ(G.numberOfEdges(), m);
```

Comment:
Please check whether edge sets are identical or at least add a TODO (I've an algorithm for that, but can create PR only in three weeks)

Reply:
Right, I've added those checks in the tests.